webpack.config.js

```diff
@@ -3,8 +3,10 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 const DotenvPlugin = require('dotenv-webpack');
 
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
looks good for now. I think NODE_ENV automatically is set to development in webpack (or at least it is in CRA) so may be more reliable/easy to check for development variable.

Btw does this need REACT_APP in the variable to work?

Reply:
this one doesnt need the react app prepended